renku/cli/dataset.py

```diff
@@ -883,3 +907,42 @@ def update(short_names, creators, include, exclude, ref, delete, external):
         progress_context=progress_context
     )
     click.secho('OK', fg='green')
+
+
+@dataset.command('mv')
+@click.argument('source-dataset')
+@click.argument('target-dataset')
+@click.option(
+    '-I',
+    '--include',
+    multiple=True,
+    help='Include files matching given pattern.'
```

Comment:
An example in the help text would be nice. You could show that the pattern needs to be  quoted to prevent shell expansion.

Reply:
So far the convention was to keep these help messages short. An example is already in the dataset documentation. We also have an issue to improve UX regarding help in commands: https://github.com/SwissDataScienceCenter/renku-python/issues/1069.